samples/ibm-samples/ffdl-seldon/ffdl_pipeline.ipynb

```diff
@@ -88,17 +103,26 @@
    "source": [
     "import kfp\n",
     "from kfp import compiler\n",
-    "import kfp.dsl as dsl\n",
-    "import kfp.notebook\n",
-    "import kfp.gcp as gcp\n",
+    "import kfp\n",
+    "from kfp import components\n",
+    "from kfp import dsl\n",
+    "from kfp import notebook\n",
+    "\n",
+    "# Run client with KUBEFLOW_PIPELINE_LINK if this notebook server\n",
+    "# is running on localhost without enterprise gateway.\n",
+    "\n",
+    "# KUBEFLOW_PIPELINE_LINK = ''\n",
+    "# client = kfp.Client(KUBEFLOW_PIPELINE_LINK)\n",
+    "\n",
+    "client = kfp.Client()\n",
     "\n",
-    "client = kfp.Client(KUBEFLOW_PIPELINE_LINK)\n",
+    "\n",
+    "EXPERIMENT_NAME = 'FfDL-Seldon Experiments'\n",
     "\n",
     "# Uncomment the below line if you want to create an experiment, \n",
     "# else we will assume the EXPERIMENT_NAME is already exist.\n",
     "\n",
-    "# exp = client.create_experiment(name=EXPERIMENT_NAME)\n",
-    "\n",
+    "#exp = client.create_experiment(name=EXPERIMENT_NAME)\n",
```

Comment:
`create_experiment` now now works the same as `get_experiment` if the experiment exists.

Reply:
@Ark-kun fixed this then - should just be _create_experiment_